src/components/RuleItem.vue

```diff
@@ -206,8 +211,12 @@ async function editSafetyStock() {
     },
     {
       text: translate('Update'),
-      handler: async (data) => {
+      handler: async (data: any) => {
         if(data.safetyStock) {
+          if(data.safetyStock < 0) {
```

Comment:
This check for safetyStock value is redundant. We can move this check above the parent `if`.